"latest" command with no versions causes panic

If the `changie latest` command is run in a project that contains no versions, the command panics.

reproduction:
```
tmp=$(mktemp -d) && cd $tmp && changie init && changie latest; rm -rf $tmp

panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/miniscruff/changie/cmd.latestPipeline(0x7bb040, 0x9a74d0, 0x74f6ab, 0x4, 0x74f64b, 0x4)
	/home/user/src/go/src/github.com/miniscruff/changie/cmd/latest.go:84 +0x785
github.com/miniscruff/changie/cmd.runLatest(0x971600, 0x9a74d0, 0x0, 0x0, 0x0, 0x0)
	/home/user/src/go/src/github.com/miniscruff/changie/cmd/latest.go:40 +0x3d
github.com/spf13/cobra.(*Command).execute(0x971600, 0x9a74d0, 0x0, 0x0, 0x971600, 0x9a74d0)
	/home/user/src/go/src/github.com/spf13/cobra/command.go:850 +0x47c
github.com/spf13/cobra.(*Command).ExecuteC(0x971de0, 0x74f1a1, 0x1, 0x74f410)
	/home/user/src/go/src/github.com/spf13/cobra/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
	/home/user/src/go/src/github.com/spf13/cobra/command.go:895
github.com/miniscruff/changie/cmd.Execute(...)
	/home/user/src/go/src/github.com/miniscruff/changie/cmd/root.go:19
main.main()
	/home/user/src/go/src/github.com/miniscruff/changie/main.go:14 +0x9d
```

--- a/changes/unreleased/Fixed-20210101-142311.yaml
+++ b/changes/unreleased/Fixed-20210101-142311.yaml
@@ -1,0 +1,4 @@
+kind: Fixed
+body: Latest command when no releases exist will return v0.0.0 instead of panicking
+custom:
+  Issue: "33"

--- a/cmd/latest.go
+++ b/cmd/latest.go
@@ -53,12 +53,12 @@ func latestPipeline(afs afero.Afero) (string, error) {
 		return "", err
 	}
 
-	allVersions := make([]*semver.Version, 0)
-
 	fileInfos, err := afs.ReadDir(config.ChangesDir)
 	if err != nil {
 		return "", err
 	}
+
+	allVersions := make([]*semver.Version, 0)
 
 	for _, file := range fileInfos {
 		if file.Name() == config.HeaderPath || file.IsDir() {
@@ -73,6 +73,11 @@ func latestPipeline(afs afero.Afero) (string, error) {
 		}
 
 		allVersions = append(allVersions, v)
+	}
+
+	// if no versions exist default to v0.0.0
+	if len(allVersions) == 0 {
+		return "v0.0.0\n", nil
 	}
 
 	sort.Sort(sort.Reverse(semver.Collection(allVersions)))

--- a/cmd/latest_test.go
+++ b/cmd/latest_test.go
@@ -72,4 +72,13 @@ var _ = Describe("Latest", func() {
 		_, err := latestPipeline(afs)
 		Expect(err).NotTo(BeNil())
 	})
+
+	It("returns v0.0.0 if no versions found", func() {
+		// include just the header
+		_, err := afs.Create("chgs/head.tpl.md")
+		Expect(err).To(BeNil())
+		res, err := latestPipeline(afs)
+		Expect(err).To(BeNil())
+		Expect(res).To(Equal("v0.0.0\n"))
+	})
 })

